schema/proto3/subnet.proto

```diff
@@ -28,35 +28,25 @@ message SubnetConfiguration {
     uint32 revision_number = 2;
 
     string id = 3;
-    string project_id = 4;
-    string vpc_id = 5;
-    string name = 6;
-    string cidr = 7;
-    int64 tunnel_id = 8;
+    NetworkType network_type = 4;  // Migrating this to subnet level
```

Comment:
We see quite a few breaking changes (which is expected for this feature). Pushing the new schema to the alcor/master will have side effects though because other services are forced to consume it. 

Can we do this? ACA pointing to the er1cthe0ne:design/routing branch so that you could unblock for L3 development. @er1cthe0ne

Reply:
actually, the current schema change has zero breaking change to Alcor as added new fields or re-order existing fields won't break things. Let me know if that make sense.